versions/1/data_stream/fields/fields.spec.yml

```diff
@@ -86,11 +86,8 @@ spec:
         type: string
         enum:
           - ecs
-      dynamic_template:
-        description: Indication that this field represents an explicitly named dynamic template
-        type: boolean
```

Comment:
`dynamic_template` is part of a GA version of the spec, included in version 1.0.0. I am not sure if we can remove it at this point. @ruflin @exekias thoughts?

You can also mark it as deprecated by now.
```
      dynamic_template:
        description: Indication that this field represents an explicitly named dynamic template
        type: boolean
        deprecated: true # Define the dynamic templates in the Elasticsearch index template mapping. 
```

Reply:
I'm torn on this one. I don't think we should break GA so deprecating is the better option. At the same time the only packages built are by us at the moment and as long as no packages uses it (also in the past) and Kibana does not have an implementation for it, we are not required to keep it. It could also be documented as a breaking change for the 1.1 version.

Reply:
Yep, no packages seem to be using this so far, and as Kibana doesn't support it, it is not useful. I am fine with including it as a breaking change, and prepare a 1.1 version (that would also include the go.mod at the root level https://github.com/elastic/package-spec/pull/213, what will allow `elastic-package` to use tagged versions of the spec).

Reply:
The field spec has `additionalProperties: true`, so the field will just be ignored anyway:

https://github.com/elastic/package-spec/blob/dc22ddb861596dc5a5f1c10e2d5fc381de3fc7c5/versions/1/data_stream/fields/fields.spec.yml#L9

Reply:
Good point. SGTM then.